c/cmake_modules/DefineOptions.cmake

```diff
@@ -232,6 +232,8 @@ if("${CMAKE_SOURCE_DIR}" STREQUAL "${CMAKE_CURRENT_SOURCE_DIR}")
   define_option(ADBC_DRIVER_BIGQUERY "Build the BigQuery driver" OFF)
 
   define_option(ADBC_INTEGRATION_DUCKDB "Build the test suite for DuckDB" OFF)
+
+  define_option(ADBC_BUILD_VCPKG "Build on Windows using vcpkg for dependencies and Visual Studio generator." OFF)
```

Comment:
Need to ensure that runtime dependencies are copied during install. Using Win32 condition causes other non vcpkg builds on windows to fail, so decided to add this specific option for the vcpkg build